LICENSE.md

```diff
@@ -6,15 +6,19 @@ Version 2.0.3
 0. Definitions
 --------------
 
+**Source Code**: The preferred form of the work for making modifications to it.
+
+**Object code**: Any non-source form of a work. 
+
 **Game**: Computer software which provides diversion or amusement. Often a form of play. 
 
-**Mod**: Software which modifies the Game, in any form (source, compiled binary, etc).
+**Mod**: Software which modifies the Game, in any form (source code, object code, etc).
 
 **Asset**: Graphic, audio, data file, or other medium used by the Mod.
 
 **User**: Anyone who that interacts with the Game or the Mod in any way.
 
-**Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
+**Dependency**: Software required for the Mod to execute, compile, or otherwise operate correctly.
```

Comment:
Given that "work" and "operate" are synonyms, I'm curious why you want to change this. Please provide a reason.